### DISCUSSION TITLE
how can the complexity of poetry be reduced?

### DISCUSSION THREAD

**Discussion by alejohern** [python-poetry/poetry#8123] [Ideas]
I wonder, 

does anyone else ever think that some of the "functionality" that os offered by poetry could be a little simplified? 
Package management for python and consistent builds in different systems, python versions, etc  is becoming a job in of itself.

**Comment by finswimmer** [comment]
We are always interested in feedback about how to improve the user experience. Can you please give us examples what is to complicated at the moment and how an improved version should look like?

fin swimmer
